src/forms/date.jsx

```diff
@@ -21,25 +21,37 @@ class DateInput extends Input {
 
   render() {
     const value = this.parseValue();
-    return <div className={this.errorClass('form-group')}>
-      <fieldset>
-        <legend>
-          <span className="form-label-bold">{this.props.label}</span>
-          { this.props.hint && <span className="form-hint">{this.props.hint}</span> }
-          { this.props.error && <span className="error-message">{this.props.error}</span> }
+    return <div className={this.errorClass('govuk-form-group')}>
+      <fieldset className="govuk-fieldset" aria-describedby={this.dateFragment('hint')} role="group">
+        <legend className="govuk-fieldset__legend">
+          <span className="govuk-label">{this.props.label}</span>
+          { this.props.hint && <span className="govuk-hint" id={this.dateFragment('hint')}>{this.props.hint}</span> }
+          { this.props.error && <span className="govuk-error-message">{this.props.error}</span> }
```

Comment:
I think the hint and error messages probably want to be outside of the legend, but there isn't really any definitive answer here: https://design-system.service.gov.uk/components/fieldset/

Reply:
I literally just copied the html from https://design-system.service.gov.uk/components/date-input/

Reply:
Ah yeah they're definitely outside legend then:
```
<legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
      <h1 class="govuk-fieldset__heading">
        What is your date of birth?
      </h1>
    </legend>
    <span id="dob-hint" class="govuk-hint">
      For example, 31 3 1980
    </span>
    <span id="dob-error" class="govuk-error-message">
      Date of birth must be in the past
    </span>
```